src/outmdsd/DjsonLogItem.cc

```diff
@@ -64,12 +66,12 @@ DjsonLogItem::ComposeSchema(
         strm << cachedInfo.first << "," << cachedInfo.second;
```

Comment:
fixed.

---
In reply to: [97008133](https://github.com/Azure/fluentd-plugin-mdsd/pull/2#discussion_r97008133) [](ancestors = 97008133)